### ISSUE TITLE
Documentation - list API's planned to never be implemented

### ISSUE THREAD

**Issue by patryk-oleniuk** [databricks/koalas#1261]
I'm missing a page that explicitly lists APIs that are planned not to be supported, and why.

This issue somewhat helped me, but I feel it should be a bit more public and kept up to date. 
https://github.com/databricks/koalas/issues/1003

Probably under `User Guide` --> `Best Practices` ? 

Thanks!

**Comment by itholic** [comment]
I agree it might better adding those APIs to official docs with reasonable reasons.

cc. @HyukjinKwon @ueshin what do you think?

**Comment by HyukjinKwon** [comment]
Yeah, I think it sounds fine.

**Comment by itholic** [comment]
@HyukjinKwon okay i'll add them to official docs while i'll work for #1003
